qutip/qip/circuit.py

```diff
@@ -31,9 +31,11 @@
 #    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-import numpy as np
+from collections.abc import Iterable
```

Comment:
Can you update this import according to #1003 ?

Reply:
I'm a bit confused. In my python version 3.7.3, this warning occurs when I run
`from collections import Iterable`
but doesn't when I run
`from collections.abc import Iterable`
It seems to me that the second one is the up-to-date one?

Reply:
@nathanshammah  Thanks for the suggestions. I have corrected all other doc formats. I think the importation I used here `from collections.abc import Iterable` should be the up-to-date one.